package.json

```diff
@@ -115,6 +115,7 @@
     "prebuild:js": "rm -rf assets/css/* assets/js/*",
     "build:js": "wp-scripts build",
     "build:plugin": "wp dist-archive . $(pwd)/build/web-stories --create-target-dir",
+    "build:storybook": "build-storybook -c .storybook -o build/storybook --quiet",
```

Comment:
That means `npm run build` will also build the storybook which might be unexpected. Otherwise this works for me.

Reply:
I think that's alright